src/tests/Microsoft.PowerFx.Core.Tests/ExpressionTestCases/Sqrt_ConsistentOneColumnTableResult.txt

```diff
@@ -0,0 +1,13 @@
+﻿#SETUP: ConsistentOneColumnTableResult
+
+>> Sqrt([1, 529, 144, 324])
+[1,23,12,18]
+
+>> Sqrt([])
+[]
+
+>> Sqrt([9, -1])
```

Comment:
this looks like the same test and result as in sqrt.txt ?

Reply:
Yep as Sqrt already implements arrays with "Value"